tests/run/t6476.scala

```diff
@@ -0,0 +1,23 @@
+object Test {
+  def main(args: Array[String]): Unit = {
+    val person = "Alice"
+    println(s"\"Hello\", $person")
+    println(s"""\"Hello\", $person""")
+
+    println(f"\"Hello\", $person")
+    println(f"""\"Hello\", $person""")
+
+    println(raw"\"Hello\", $person")
```

Comment:
Questions: Now that we have `$"` as way to escape `"` in any string interpolator, do we really need to special case `\"` in raw interpolators? Cant we just assume that `raw"\"` is `"\"`? How can we end with a `\` in a `raw` now?

Reply:
> do we really need to special case `\"` in raw interpolators?

Maybe I misunderstand, but there's no special case or change in the `raw` interpolator. The change here is that `\"` no longer closes the string literal, but the interpretation of `\"` is unchanged (`raw"""\""""` and `s"""\""""` are the same as before).

In my opinion we should do this change change and fix https://github.com/scala/bug/issues/6476, even if we have `$"` now. This issue affected a lot of people, and will keep affecting a lot of people in the future, especially beginners (https://stackoverflow.com/questions/21086263/how-to-insert-double-quotes-into-string-with-interpolation-in-scala).

See also the (updated) PR description of https://github.com/scala/scala/pull/8830#issue-394463165.

> Cant we just assume that `raw"\"` is `"\"`?

I'm not sure what you mean by that.

> How can we end with a `\` in a `raw` now?

Using triple-quote. The 2.13 compiler will even tell you that:

```scala
scala> raw"c:\"
           ^
       error: unclosed string literal; note that `\"` no longer closes single-quoted interpolated string literals since 2.13.6, you can use a triple-quoted string instead
```

If desired, I can include this message in Scala 3 as well.